src/main/java/io/openshift/booster/BoosterIndexer.java

```diff
@@ -7,20 +7,20 @@
 
 package io.openshift.booster;
 
+import io.openshift.booster.catalog.BoosterCatalogService;
+import io.openshift.booster.catalog.spi.NativeGitBoosterCatalogPathProvider;
+
 import java.io.OutputStream;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
-import io.openshift.booster.catalog.BoosterCatalogService;
-import io.openshift.booster.catalog.spi.NativeGitBoosterCatalogPathProvider;
-
 /**
  * Indexes a Booster catalog and adds all its contents to a ZIP file
  *
  * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
  */
-public class BoosterIndexer {
+class BoosterIndexer {
```

Comment:
This is meant to be called from the maven plugin, not sure removing the public modifier will allow that?